sssom/parsers.py

```diff
@@ -19,9 +20,11 @@
 from linkml_runtime.loaders.json_loader import JSONLoader
 from rdflib import Graph, URIRef
 
-# from .sssom_datamodel import Mapping, MappingSet
+# TODO: PR comment: where matchtypeenum? can't find sssomschema, Mapping, or MappingSet. only MappingSetDataFrame
```

Comment:
@matentzn I rebased from `master`, but I have 2 issues that need fixing that I need help with:
## Issue 1: `ModuleNotFoundError: No module named 'sssom_schema'`
I saw that what I was previously importing from `.sssom_datamodel` is now imported from `sssom_schema`. I assume this comes with a dependency. So I ran `tox` to install the latest dependencies, but even the test suite failed on this very issue. Here's the full output from `.tox`:

[tox-output.txt](https://github.com/mapping-commons/sssom-py/files/9105522/tox-output.txt)

## Issue 2: Where is `MatchTypeEnum`?
I found `MatchTypeEnum` useful and I want it back. I'm assuming this is now in `sssom_schema`, but I need to resolve issue (1) before I can get it. Or, if it's not there, I can add MatchTypeEnum back somewhere. Here is the definition that I pulled from a previous commit:

```py
class MatchTypeEnum(EnumDefinitionImpl):

    Lexical = PermissibleValue(text="Lexical", description="Lexical match")
    Logical = PermissibleValue(text="Logical", description="Logical match")
    HumanCurated = PermissibleValue(
        text="HumanCurated", description="Match based on human expert opinion"
    )
    Complex = PermissibleValue(
        text="Complex", description="Match based on a variety of different strategies"
    )
    Unspecified = PermissibleValue(text="Unspecified", description="Unknown match type")
    SemanticSimilarity = PermissibleValue(
        text="SemanticSimilarity",
        description="Match based on close semantic similarity",
    )

    _defn = EnumDefinition(
        name="MatchTypeEnum",
    )
```

Reply:
Nico, please correct me if I'm wrong.
Joe:
First check: Have you pulled from the main branch?
Issue 1: replace `from .sssom_datamodel import Mapping, MappingSet` to `from sssom_schema import Mapping, MappingSet` I think this should fix it.

Issue 2: `MatchTypeEnum` is now replace by [SEMAPV](https://github.com/mapping-commons/sssom-py/blob/da1256830f4f2dbe4eb70fbe78945074e882ca52/sssom/constants.py#L122). 

I hope this helps!

Reply:
I got tox working now, thanks. For issue 2, I am going to close this and open a new comment.

Reply:
@hrshdhgd Thanks so much for responding to this!

Issue 1
Yeah I did get the latest code. I did a `rebase` of my branch from `master`. I pinged Nico and he asked me to just remove my `.tox` dir and run the command again. that worked.

Issue 2
Very helpful, thanks! This is exactly what I was looking for. I tried various searches to see where this was moved to but couldn't find it; search fail on my part.

I will update my code to use this, though I think the only SNOMED::ICD mapping that I was able to ascertain from the  mapping file that SNOMED published was "UnspecifiedMatching", but I should probably double check.